### DISCUSSION TITLE
simpleaudio (1.0.4) not supporting PEP 517 builds

### DISCUSSION THREAD

**Discussion by karkir0003** [python-poetry/poetry#9418] [General]
I'm trying to use poetry to install the `soundata` library. However, when I install it, I'm getting an error message like:

```
Note: This error originates from the build backend, and is likely not a problem with poetry but with simpleaudio (1.0.4) not supporting PEP 517 builds. You can verify this by running 'pip wheel --no-cache-dir --use-pep517 "simpleaudio (==1.0.4)"'.
```
When I try running ` pip wheel --no-cache-dir --use-pep517 "simpleaudio (==1.0.4)"`, I'm getting further issues. 

This is the GitHub action that's failing in my team's project when we add `soundata` as a dependency for context: https://github.com/DSGT-DLP/Deep-Learning-Playground/actions/runs/9072655948/job/24928475916?pr=1181

Could someone please advise? Thanks!

**Comment by finswimmer** [comment]
Hey,

there is no wheel file for `simpleaudio` for your platform and python version. This is why it needs to be build from source. This seems to fail because of a missing dependency in the system. See https://simpleaudio.readthedocs.io/en/latest/installation.html#linux-dependencies

fin swimmer
